### PR TITLE
Added support for providing the original image in the summary

### PR DESCRIPTION
**Pull request by Pchelolo**
Pending today's media wiki deployment.

### CODE REVIEW

File: v1/trending.yaml [hunk 1]
```diff
@@ -64,6 +78,9 @@ definitions:
       thumbnail:
         description: Lead image for the article
         $ref: '#/definitions/thumbnail'
+      originalimage:
+        description: The original of the lead image for the atricle
```
Comment (berndsi) [inline v1/trending.yaml]: typo in article

File: v1/related.yaml [hunk 2]
```diff
@@ -102,6 +116,9 @@ definitions:
             thumbnail:
               description: Lead image for the article
               $ref: '#/definitions/thumbnail'
+            originalimage:
+              description: The original of the lead image for the atricle
```
Comment (berndsi) [inline v1/related.yaml]: typo in article

File: v1/feed.yaml [hunk 3]
```diff
@@ -126,8 +140,11 @@ definitions:
         type: string
         description: Article title as it should be presented to the user
       thumbnail:
-        description: Lead image for the article
+        description: Thumbnail for the lead image for the article
         $ref: '#/definitions/thumbnail'
+      originalimage:
+        description: The original of the lead image for the atricle
```
Comment (berndsi) [inline v1/feed.yaml]: typo in article

**Comment by Pchelolo** [comment]
Mediawiki dependency has been deployed, so this can now be merged and deployed as well

**Pull request merged**